lineapy/cli/cli.py

```diff
@@ -164,7 +165,8 @@ def file(
         name=artifact_name,
         date_created=artifact.date_created,  # type:ignore
     )
-    logger.info(api_artifact.get_code())
+    # logger.info(api_artifact.get_code())
+    print(api_artifact.get_code())
```

Comment:
logger redirects stdout to stderr and changing it caused other tests to fail because outputs dont match. moving back to print here for just notebook and file cli commands.